added German language

crispy-cookie opened the pull request:
Are special characters like ä,ü,ö,ß allowed or should I change them to ae,ue,oe,ss? (check line 17 & 20)
I couldn't build a .jar file, so I couldn't test it.

Commit:
added German

are special characters like ä,ü,ö,ß accepted?

Commit:
forgot to change PrefixText

Are special characters like ä,ö,ü,ß allowed?

UltraBlackLinux commented:
Hey, thanks for your contribution - I just set up the gradle workflows, so you can test it out.
Please leave it as white-/blacklist though, it makes a bit more sense for me.

Commit:
Merge branch 'UltraBlackLinux:main' into main

Commit:
changed general.savingOptions.@Tooltip

changed general.savingOptions.@Tooltip[2] to Whitelist
changed general.savingOptions.@Tooltip[3] to Blacklist

UltraBlackLinux commented:
Thank you ;)

You know, I am German, too xD

You can download the updated version from the GH actions
